src/App.js

```diff
@@ -1,16 +1,62 @@
-import React from 'react';
+import React, {Component} from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+
+let storage = window.localStorage;
+let storageList = [];
+if (storage.list !== undefined){
+	storageList = JSON.parse(storage.list);
+}
```

Comment:
You're a strange one, Tommy. 🔥
I'd recommend not using local storage unless you're comfortable with the other material first